### ISSUE TITLE
Regarding the type of bbox

### ISSUE THREAD

**Issue by rp7sv** [opacity-black/RGBT_toolkit#2]
Great job, thank you very much for providing the open-source code.
I wanted to ask if the code supports the 'center' (xywh) bbox type? When I attempted to register the tracker and changed the bbox_type to 'center', I received the following error.
![微信图片_20240509144303](https://github.com/opacity-black/RGBT_toolkit/assets/79622142/d3ebec63-6e61-41d5-a1c2-9e18babdfe81)

**Comment by opacity-black** [comment]
Yes, it supports that. I think changing it to “xywh” will work. I will add this one sometime in the future.

**Comment by rp7sv** [comment]
Thank you very much for your prompt reply.
As you mentioned, I debugged the `'bbox_type_trans'` method in `utils.py` and found that the input bbox types are ultimately converted to `bbox_type_new==ltwh` for calculation. The `xywh_2_ltwh` function you provided is perfect for converting input types to 'center', so when registering the tracker, I should write `xywh`.
However, I encountered a new problem. The MPR and MSR produced by my tracker are unusually low, and the code cannot generate various result images. Could you please point out where I might have overlooked? The error message is as follows:
![1](https://github.com/opacity-black/RGBT_toolkit/assets/79622142/0c6d8674-c22c-4241-aff9-8e8325b199e6)

**Comment by rp7sv** [comment]
I saw the following information on the RGBT234 official website
![2](https://github.com/opacity-black/RGBT_toolkit/assets/79622142/44d012b3-438c-4e6d-a65e-a2feb8e37322)

**Comment by opacity-black** [comment]
To make it easier to deal with, I converted all bboxes to a uniform format `ltwh`, and actually my ground-truth files came from the officially provided. 
I haven't encountered the above error, if you don't mind, send me the result file and I'll try to fix it. Or you can provide more detailed information.
email: 2690598117@qq.com
